app/build.gradle

```diff
@@ -1,6 +1,5 @@
 buildscript {
-  ext.gradleVersion = '7.5.1'
```

Comment:
Does removing this mean we get the most recent version? Should we fix it to 8.0 instead?